lib/purchase_processor/orchestrator/transactions/purchase_transaction_orchestrator.rb

```diff
@@ -22,7 +22,7 @@ def self.create_transaction( provider, attributes, request_attributes = nil )
           # For Stripe
           merge_tokens( attributes )
 
-          unless subscription_attributes.blank? && existing_local_product( customer, product ).present? && orchestrator_response.is_a?( OrchestratorError )
+          unless subscription_attributes.blank? && existing_local_product( customer, product ).present?
```

Comment:
why remove the orchestratorError check here, is there ever a time that one would be present / returned at this point?

Reply:
the orchestrator response is still `nil` at this point, so the check isn't actually doing anything